Application/EdFi.Ods.WebService.Tests/App.config

```diff
@@ -20,10 +20,7 @@
     <!-- Uncomment WebDriver setting to see UI tests run in Firefox-->
     <!--<add key="WebDriver" value="Firefox"/>-->
     <!-- Sql Server 2008 by Default-->
-    <add key="SqlServerDataDirectory" value="C:\Program Files\Microsoft SQL Server\MSSQL10_50.MSSQLSERVER\MSSQL\DATA" />
-    <add key="QueuePrefix" value="EdFiOds" />
-    <add key="QueueAutoCreate" value="1" />
-    <add key="CommitUploadCommandMessageEndPoint" value="localhost" />
+    <add key="SqlServerDataDirectory" value="C:\Program Files\Microsoft SQL Server\MSSQL10_50.MSSQLSERVER\MSSQL\DATA" />        
     <add key="StartOperationCommandMessageEndpoint" value="localhost" />
```

Comment:
Also for bulk

Reply:
yes removed "StartOperationCommandMessageEndpoint"